mixin/package.go

```diff
@@ -134,6 +136,12 @@ func addPackage(pkg string, build bool) (string, error) {
 		return "", err
 	}
 
+	// if the bundleName argument is non-empty use this instead of
```

Comment:
Perhaps getPackageRepo should be renamed getPackageFromRepo, when I saw this code I was thinking we could name the passed in parameter as "bundle" and only call getPackageRepo if we needed a name (i.e. were passed in a empty string) until I checked.